ux.symfony.com/.twig-cs-fixer.dist.php

```diff
@@ -1,5 +1,44 @@
 <?php
 
-$config = new TwigCsFixer\Config\Config();
+use TwigCsFixer\Rules\Delimiter\BlockNameSpacingRule;
+use TwigCsFixer\Rules\File\DirectoryNameRule;
+use TwigCsFixer\Rules\File\FileExtensionRule;
+use TwigCsFixer\Rules\File\FileNameRule;
+use TwigCsFixer\Rules\Function\IncludeFunctionRule;
+use TwigCsFixer\Rules\Punctuation\TrailingCommaSingleLineRule;
+use TwigCsFixer\Rules\String\SingleQuoteRule;
+use TwigCsFixer\Rules\Variable\VariableNameRule;
+use TwigCsFixer\Rules\Whitespace\EmptyLinesRule;
+use TwigCsFixer\Rules\Whitespace\TrailingSpaceRule;
+use TwigCsFixer\Ruleset\Ruleset;
+use TwigCsFixer\Rules\Whitespace\BlankEOFRule;
+use TwigCsFixer\Standard\Twig;
+
+$uxRuleset = (new Ruleset())
+    ->addStandard(new Twig())
+    // Symfony standard (customized for Symfony UX)
+    ->addRule(new DirectoryNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components']))
+    ->addRule(new FileNameRule(baseDirectory: 'templates', ignoredSubDirectories: ['bundles', 'components', 'demos/live_memory/components'], optionalPrefix: '_'))
+    ->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'templates/components'))
+    ->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'templates/components'))
+    ->addRule(new DirectoryNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
+    ->addRule(new FileNameRule(case: DirectoryNameRule::PASCAL_CASE, baseDirectory: 'demos/live_memory/components'))
+    ->addRule(new FileExtensionRule())
+    // TwigCsFixer standard (customized for Symfony UX)
+    ->addRule(new BlankEOFRule())
+    ->addRule(new BlockNameSpacingRule())
+    ->addRule(new EmptyLinesRule())
+    ->addRule(new IncludeFunctionRule())
+    ->addRule(new SingleQuoteRule())
+    ->addRule(new TrailingSpaceRule())
+    ->addRule(new TrailingCommaSingleLineRule())
+    ->overrideRule(new VariableNameRule(VariableNameRule::SNAKE_CASE, '_'))
```

Comment:
Maybe this should be the "default" in the Twig Standard

The standard say `Use lower cased and underscored variable names:` cf https://twig.symfony.com/doc/3.x/coding_standards.html

Does allowing a `_` prefix is "standard" for twig variable ?

Reply:
I'm not sure, to be honest, but it's a habit I've had for a long time 😊

Reply:
I'm curious, when do you use `_foo` ?

Reply:
In a given template where you have "categories" (the list) and "category" (the current one) for instance

I often do stuff like that

```twig
{% for _category in categories %}

    <a href="#" class="{{ _category.id == category.id ? 'current' }}">{{ category.name }}</a>

{% endfor %}  
```